home-manager/home-manager

```diff
@@ -490,6 +523,21 @@ while [[ $# -gt 0 ]]; do
             HOME_MANAGER_CONFIG="$1"
             shift
             ;;
+        --flake)
+            setFlakeAttribute "$1"
+            shift
+            ;;
+        --recreate-lock-file|--no-update-lock-file|--no-write-lock-file|--no-registries|--commit-lock-file)
+            PASSTHROUGH_OPTS+=("$opt")
+            ;;
+        --update-input)
+            PASSTHROUGH_OPTS+=("$opt" "$1")
+            shift
+            ;;
+        --override-input)
+            PASSTHROUGH_OPTS+=("$opt" "$1" "$2")
+            shift 2
+            ;;
```

Comment:
Are they supported by `nixos-rebuild`?

Reply:
Yes, I didn't know that either and only implemented it when I saw the options in the nixos-rebuild.sh script. They're undocumented, so it's pretty easy to miss them.